src/domains/shared-app/api/session-wallet/v1/_fragment_policiy_chk_jwt.tpl.xml

```diff
@@ -11,8 +11,11 @@
             <key>{{wallet-session-jwt-signing-key}}</key>
         </issuer-signing-keys>
         <required-claims>
-          <claim name="walletId" match="all">
-            <value>@((string)context.Variables.GetValueOrDefault("walletId",""))</value>
+          <claim name="userId" match="all">
```

Comment:
tnx to remove `walletId`

Reply:
I need to restore the wallet token instead i think. It is useful for api vs pm. It will be removed when pm will be dismissed